Gruntfile.js

```diff
@@ -21,7 +20,7 @@ module.exports = function(grunt) {
                 noarg: true,
                 regexp: true,
                 undef: true,
-                unused: true,
+                unused: 'strict',
```

Comment:
What does this change do? Does this have to do with error being thrown when you have an unused variable? I don't notice a difference in that in changing it from true to 'strict'

Reply:
This checks for params in the function definition, and if they are actually used in the function itself. See [here](https://jshint.com/docs/options/#unused).

Reply:
Ohh I see, thank you! Personally this linter gets really tedious to deal with when developing and creating different variables, etc. but now I know where to turn it off when i'm just developing and turn it back on later. thanks!